src/sentry/api/endpoints/rule_snooze.py

```diff
@@ -0,0 +1,90 @@
+import datetime
+
+from rest_framework import serializers, status
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry import audit_log
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases.project import ProjectAlertRulePermission, ProjectEndpoint
+from sentry.api.exceptions import ConflictError
+from sentry.api.serializers import serialize
+from sentry.incidents.models import AlertRule
+from sentry.models import Rule, RuleSnooze
+
+
+class RuleSnoozeSerializer(serializers.Serializer):
```

Comment:
You can base this off of `CamelSnakeSerializer` then define the columns as `user_id` etc and it'll automatically translate values from camel -> snake for you